src/main/java/io/supertokens/thirdparty/ThirdParty.java

```diff
@@ -164,8 +161,8 @@ public static UserPaginationContainer getUsers(Main main,
     }
 
     public static UserInfo[] getUsersByEmail(Main main, @Nonnull String email) throws StorageQueryException, InvalidInputException {
-        if (!EmailValidator.isValid(email)) {
-            throw new InvalidInputException("email is invalid");
+        if (email.equals("")) {
```

Comment:
We should not even do this check. No validation of user input means no validation.